src/main/java/athena/TimeAllocate.java

```diff
@@ -0,0 +1,57 @@
+package athena;
+
+import athena.task.Task;
+import athena.task.taskfilter.FlexibleTimeFilter;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+
+public class TimeAllocate {
```

Comment:
Can we call this `TimeAllocator`? A noun instead of a verb since it's an object.

Reply:
Ok will change soon